plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHiveMetastoreClient.java

```diff
@@ -351,14 +353,16 @@ public void dropRole(String role)
     public boolean grantPrivileges(PrivilegeBag privilegeBag)
             throws TException
     {
-        return client.grant_privileges(privilegeBag);
+        return client.grant_revoke_privileges(new GrantRevokePrivilegeRequest(GRANT, privilegeBag)).isSuccess();
```

Comment:
What should happen if it is not successful? Show we retry or fail? Do we check that? I don't see any check for this in `io.trino.plugin.hive.metastore.thrift.ThriftHiveMetastore#revokeTablePrivileges`

Reply:
Why do we need this change at all? Is it only for symmetry with `revokePrivileges`?

Reply:
> Why do we need this change at all

`Client#grant_privileges` is deprecated so we move to the new API. And yet it would be symmetric with revokePrivileges.

Reply:
> Show we retry or fail.
We retry in case of any failures. 

But here we just leave it as it is. Like we do for `dropPartitions`